operator/apis/platform/v1alpha1/common_types.go

```diff
@@ -166,3 +166,33 @@ const (
 	// ScannerAutoScalingDisabled means that scanner autoscaling should be disabled.
 	ScannerAutoScalingDisabled AutoScalingPolicy = "Disabled"
 )
+
+// Monitoring defines settings for monitoring endpoint.
+type Monitoring struct {
+	// Expose the monitoring endpoint. A new service, "monitoring",
+	// with port 9090, will be created as well as a network policy allowing
```

Comment:
does this have to be 9090?

Reply:
It doesn't have to be, but it's a very common choice for Prometheus. I don't think we gain much by making the port configurable, and we already hard code it for Central.